charts/istio-monitors/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```

Comment:
I'm not a huge fan of this chart being introduced as Istio things are now scattered across different places. Is there an easy way to extend the Istio chart and pass these objects there instead? The istio chart does not expose such an interface unfortunately, so we'd need something that's built into Helm itself instead.

Reply:
My understanding but we didn't use this pattern before is that a "subchart" type re-use of the off the shelf chart in combination with our own parent (or possibly our own parent and our own sub "monitoring" chart) would be what most people would do. See: https://helm.sh/docs/chart_template_guide/subcharts_and_globals/

Reply:
Let me try getting that to work.